apps/site/scripts/orama-search/process-documents.mjs

```diff
@@ -0,0 +1,86 @@
+import { sep as posixSep, basename } from 'node:path/posix';
+import { sep as winSep } from 'node:path/win32';
+
+import { slug } from 'github-slugger';
+import matter from 'gray-matter';
+
+// Note: We don't remove Markdown Headers delimiters as they're useful for delimiting sections
+// TODO(@avivkeller): Use remark
+const parseRichTextIntoPlainText = richText =>
```

Comment:
Shouldn't we use remark/unist to simply give the plain text interpretation? I mean, regexes work, but sound like quite the amount of arbitrary regexes. And IMO going simply through text nodes seems to be simpler/easier.